### PR TITLE
[Backend] Thumbnail creator from `bounding_box` data

### PR DESCRIPTION
**Pull request by asyafalni**
## Type of changes

To support nodeflux analytics with multiple object detection, backend translates bounding_box data into real image thumbnails, therefore frontend can display result in better presentation

Please check the type of change your PR introduces:
- [ ] Bugfix
- [x] Enhancement
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [x] Documentation content changes
- [ ] Other (please describe): 

## What is the current behaviour?

No thumbnails data

## What is the new behavior?

field thumbnails on service response,
```json
{
    "message": string,
    "ok": boolean,
    "service_data": object, // json data from service response
    "thumbnails": string array, // png in base64 data with data uri scheme
}
```
 
Screenshot from 2021-11-10 16-48-08

## Checklist

- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules
- [x] I have filled the `[Unreleased]` [Changelog](https://github.com/nodefluxio/weber/blob/main/CHANGELOG.md

## Security Pentester API Checklist
- [ ] IDOR
- [ ] Malicious Script Injection
- [ ] SQL Injection

## Does this introduce a breaking change?

- [ ] Yes
- [x] No

## Other information

Thumbnail example
download (2)

### CODE REVIEW

**Review by krismp — APPROVED** [review]

**Comment by krismp** [comment]
Conflict @asyafalni

**Review by krismp — COMMENTED** [review]
Changelog nya @asyafalni

**Review by gregoriusjimmy — DISMISSED** [review]

**Comment by asyafalni** [comment]
> Conflict @asyafalni

otw fix

**Comment by asyafalni** [comment]
> Changelog nya @asyafalni

done

File: backend/controllers/analytic.go [hunk 1]
```diff
@@ -166,4 +173,69 @@ func GetResultFaceMatchEnrollment(service models.Service, input models.RequestDa
...
+	return img, cfg, err
+}
+
+func CropImage(img image.Image, cfg image.Config, bbox models.BoundingBox) (string, string, error) {
+	var Left int = int(bbox.Left * float64(cfg.Width))
+	var Top int = int(bbox.Top * float64(cfg.Height))
+	var Width int = int(bbox.Width * float64(cfg.Width))
+	var Height int = int(bbox.Height * float64(cfg.Height))
+	
+	fmt.Println("Width:", cfg.Width, "Height:", cfg.Height)
```
Comment (lazuardyk) [inline backend/controllers/analytic.go:203]: Ini diapus aja kali ya printnya?

Comment (asyafalni) [reply backend/controllers/analytic.go:203]: sabi

**Review by krismp — APPROVED** [review]

**Review by lazuardyk — APPROVED** [review]
mantapppp

**Pull request merged**